files/zh-cn/mdn/community/contributing/our_repositories/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: MDN Web 文档仓库
+slug: MDN/Community/Contributing/Our_repositories
+---
+
+{{MDNSidebar}}
+
+[MDN Web 文档](/)是一个复杂的项目，有很多不同的部分。熟悉项目的不同仓库（repository）是个好主意。本文档旨在帮助你找到在为 MDN Web 文档项目中不同部分做出贡献时可能需要的不同仓库。
+
+## 仓库层级
+
+### 一级
+
+这些仓库中的代码是 MDN Web 文档项目的核心，并运行在 Mozilla 拥有的域名上。
+
+- [mdn/content](https://github.com/mdn/content)
+- [Yari](https://github.com/mdn/yari)
+- [rumba](https://github.com/mdn/rumba)
+- [browser-compat-data](https://github.com/mdn/browser-compat-data)
+- [interactive-examples](https://github.com/mdn/interactive-examples)
+- [bob](https://github.com/mdn/bob)
+
+一级项目至少应该有 3 名成员，其中至少有两名具有管理员权限。
+
+### 二级
+
+这些仓库主要集中于支持内容，例如代码示例、MDN Web 文档学习区、本地化和示例项目。例如：
+
+- [dom-examples](https://github.com/mdn/dom-examples)
+- [translated-content](https://github.com/mdn/translated-content)
+- [learning-area](https://github.com/mdn/learning-area)
+
+二级项目至少应该有 2 名成员，其中至少有一名具有管理员权限。
+
+### 三级
+
+这些仓库用于项目规划、记录项目本身和社区参与。例如：
+
+- [mdn-community](https://github.com/mdn/mdn-community)
+- [mdn/mdn](https://github.com/mdn/mdn)
+- [content-team-projects](https://github.com/mdn/content-team-projects).
```

Comment:
since previous unordered list have no period ended, I believe it's a typo

```suggestion
- [content-team-projects](https://github.com/mdn/content-team-projects)
```